### PR TITLE
configure hibernate, add models, dao and service

### PR DESCRIPTION
**Pull request by KovalAlexey**

### CODE REVIEW

**Review by aleksandr-hrankin — APPROVED** [review]

File: src/main/java/com/dev/cinema/dao/impl/MovieDaoImpl.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+@Dao
+public class MovieDaoImpl implements MovieDao {
+    @Override
+    public Movie add(Movie movie) {
+        Transaction transaction = null;
+        Session session = null;
+        try {
+            session = HibernateUtil.getSessionFactory().openSession();
+            transaction = session.beginTransaction();
+            transaction.commit();
```
Comment (KseniiaMakarova) [inline src/main/java/com/dev/cinema/dao/impl/MovieDaoImpl.java]: where do we save the entity?

File: src/main/java/com/dev/cinema/service/impl/MovieServiceImpl.java [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+import com.dev.cinema.lib.Inject;
+import com.dev.cinema.lib.Service;
+import com.dev.cinema.model.Movie;
+import com.dev.cinema.service.MovieService;
+import java.util.List;
+
+@Service
+public class MovieServiceImpl implements MovieService {
+    @Inject
+    MovieDao movieDao;
```
Comment (KseniiaMakarova) [inline src/main/java/com/dev/cinema/service/impl/MovieServiceImpl.java]: access modifier

File: src/main/java/com/dev/cinema/util/HibernateUtil.java [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    private static SessionFactory sessionFactory = initSessionFactory();
+
+    private HibernateUtil() {
+    }
+
+    private static SessionFactory initSessionFactory() {
+        try {
+            return new Configuration().configure().buildSessionFactory();
+        } catch (Exception e) {
+            throw new DataProcesingException("Error creating Session Factory", e);
```
Comment (KseniiaMakarova) [inline src/main/java/com/dev/cinema/util/HibernateUtil.java]:
```suggestion
            throw new RuntimeException("Error creating Session Factory", e);
```
this Exception has a different nature

File: src/main/resources/hibernate.cfg.xml [hunk 4]
```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE hibernate-configuration PUBLIC
+        "-//Hibernate/Hibernate Configuration DTD 3.0//EN"
+        "http://hibernate.org/dtd/hibernate-configuration-3.0.dtd">
+<hibernate-configuration>
+    <session-factory>
+        <property name="hibernate.dialect">org.hibernate.dialect.MySQL5Dialect</property>
+        <property name="connection.url">jdbc:mysql://localhost/cinema?serverTimezone=UTC</property>
```
Comment (KseniiaMakarova) [inline src/main/resources/hibernate.cfg.xml]: does it use 3306 by default?

**Review by KseniiaMakarova — CHANGES_REQUESTED** [review]

File: src/main/resources/hibernate.cfg.xml (hunk 4 above)
Comment (KovalAlexey) [reply src/main/resources/hibernate.cfg.xml]: my local database is set to 3306 by default, done

File: src/main/java/com/dev/cinema/util/HibernateUtil.java (hunk 3 above)
Comment (KovalAlexey) [reply src/main/java/com/dev/cinema/util/HibernateUtil.java]: I knew it(
done

File: src/main/java/com/dev/cinema/service/impl/MovieServiceImpl.java (hunk 2 above)
Comment (KovalAlexey) [reply src/main/java/com/dev/cinema/service/impl/MovieServiceImpl.java]: done

File: src/main/java/com/dev/cinema/dao/impl/MovieDaoImpl.java (hunk 1 above)
Comment (KovalAlexey) [reply src/main/java/com/dev/cinema/dao/impl/MovieDaoImpl.java]: done

**Review by KseniiaMakarova — APPROVED** [review]

**Pull request merged**